.github/workflows/tests.yaml

```diff
@@ -35,6 +35,10 @@ jobs:
       run: |
         source miniforge3/bin/activate ceesd
         ./version.sh ./mirgecom/requirements.txt
+        
+        # Don't build pyopencl as that is difficult
+        cat requirements.txt.* | sed /pyopencl/d > req.txt
+        pip install -r req.txt
```

Comment:
I'm not sure I like this as the way to use the generated `requirements.txt`. I think it'll ultimately be safer to have emirge generate a new environment from scratch, and I feel that that should be the documented procedure.

Reply:
My thinking here was that emirge/mirgecom should be able to run with the "reinstalled" package versions, which is why I did this pip install before running the examples.

Imho, emirge/conda updates should be rare compared to dev packages updates (?), so we should make sure that this procedure works even without a new env.

Reply:
Ok, added a test.